networking/network_observability/network-observability-overview.adoc

```diff
@@ -0,0 +1,50 @@
+:_content-type: ASSEMBLY
+[id='network-observability-overview']
+= About Network Observability
+include::_attributes/common-attributes.adoc[]
+:context: network-observability-overview
+
+toc::[]
+
+Red Hat offers cluster administrators the Network Observability Operator to observe the network traffic for {product-title} clusters. The Network Observability uses the eBPF technology to create network flows. The network flows are then enriched with {product-title} information and stored in Loki. You can view and analyze the stored netflow information in the {product-title} console for further insight and troubleshooting.
+
+[id="dependency-network-observability"]
+== Dependency of Network Observability Operator
+
+The Network Observability Operator requires the following Operators:
+
+* Loki: You must install Loki. Loki is the backend that is used to store all collected flows. It is recommended to install the Loki Operator for the installation of Network Observability Operator.
+
+[id="optional-dependency-network-observability"]
+== Optional dependencies of the Network Observability Operator
+
+* Grafana: You can install Grafana for using custom dashboards and querying capabilities, for instance using the Grafana Operator. Red Hat does not support Grafana Operator.
+* Kafka: It provides scalability, resiliency and high availability in the {product-title} cluster. It is recommended to install Kafka using the AMQ Streams operator for large scale deployments.
+
+[id="network-observability-operator"]
+== Network Observability Operator
+
+The Network Observability Operator deploys a monitoring pipeline to collect and enrich network flows. The eBPF agent is deployed as a `daemonset` object and creates network flows. The network flows are then collected and enriched with the Kubernetes metadata, before stored in Loki.
+
+[id="no-console-integration"]
+== {product-title} console integration
+
+{product-title} console integration offers overview, topology view and Netflow tables.
```

Comment:
```suggestion
{product-title} console integration offers overview, topology view and traffic flow tables.
```
The term Netflow is removed from UI perspective hence let's not use that in our doc.

Reply:
Acked